tools/functional-tester/etcd-tester/key_stresser.go

```diff
@@ -67,16 +67,18 @@ func (s *keyStresser) Stress() error {
 
 	kvc := pb.NewKVClient(conn)
 
+	s.keyOps = 0
+
 	var stressEntries = []stressEntry{
-		{weight: 0.7, f: newStressPut(kvc, s.keySuffixRange, s.keySize)},
+		{weight: 0.7, f: newStressPut(kvc, s.keySuffixRange, s.keySize, s)},
```

Comment:
why add `s` everywhere? just change `stressFunc` to return the number of modified keys and do the atomic add in `run`? it would be less code overall